include/openssl/rsa.h

```diff
@@ -215,6 +215,88 @@ OPENSSL_EXPORT int RSA_set0_factors(RSA *rsa, BIGNUM *p, BIGNUM *q);
 OPENSSL_EXPORT int RSA_set0_crt_params(RSA *rsa, BIGNUM *dmp1, BIGNUM *dmq1,
                                        BIGNUM *iqmp);
 
+// RSA_METHOD functions
```

Comment:
RSA_METHOD wasn't deprecated before so I am leaving it as so. Since EC_KEY_METHOD was a completely new feature, we made the conscious decision to put in a deprecated section.